src/alchemlyb/estimators/mbar_.py

```diff
@@ -57,6 +57,20 @@ class MBAR(BaseEstimator, _EstimatorMixOut):
         The estimated statistical uncertainty (one standard deviation) in
         dimensionless free energy differences.
 
+    delta_h_ : DataFrame
+        The estimated dimensionless enthalpy difference between each state.
```

Comment:
Is enthalpy dimensionless? Should it be having energy unit?

Reply:
Do you want me to change the docstring? I put this since the existing documentation for free energy is:
```
    delta_f_ : DataFrame
        The estimated dimensionless free energy difference between each state.
```
and I'm treating enthalpy and entropy the same as free energy... it seems I should change entropy appropriately though.

Reply:
Ok, then I think this is fine. I will change this in a later PR.